Update_and_Start_Bot.bat

```diff
@@ -1,5 +1,11 @@
 @echo off
 
+SETLOCAL
+IF /i NOT "%~dp0"=="%temp%\" (
+ COPY /y "%~dpnx0" "%temp%\%~nx0" >nul
+ "%temp%\%~nx0"
+)
```

Comment:
Quick breakdown of what's happening here?

Reply:
it create a temp copy of the batch file, run the copy then update the original batch file

Reply:
This probably isn't enough. We need it to only start the bot if there wasn't any updates.

Reply:
Done